script/Arcueid/Day 8/08_00_ARC08_1_1.txt

```diff
@@ -13,6 +13,11 @@ I feel like my usual approach isn't going to cut it here,
 -- Page 35, Offset 19761.
 -- 「グモニー。いい朝だね、秋葉」
 // Swap EN foreign loans for different foreign loans for effect?
+// B: my confidence in this is wavering. We're already known as a French
+// group for some reason or another. 2+ French words inserted in place of
+// english loan words... I dunno about that. bai-bai isn't voiced and thus
+// much cleaner to get away with. This one is /voiced/.
+// I can be walked off this ledge.
```

Comment:
I'm still pro- loanword shuffle. French words are plenty common as loanwords in EN, and I also just don't want to have to write the sentence
> Good morning. Lovely morning..